src/vs/editor/contrib/codeAction/browser/codeActionContributions.ts

```diff
@@ -45,3 +45,15 @@ Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfigurat
 		},
 	}
 });
+
+Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfiguration({
+	...editorConfigurationBaseNode,
+	properties: {
+		'editor.codeActions.triggerOnFocusChange': {
+			type: 'boolean',
+			scope: ConfigurationScope.LANGUAGE_OVERRIDABLE,
```

Comment:
Should this be `restricted`?

Reply:
"value of this configuration will be read only from trusted sources." not entirely sure what this means? our other code actions on save aren't `restricted` iirc, but unsure if this needs or not